README.md

```diff
@@ -24,6 +24,33 @@ $ spiel demo present
 ![The first slide of the demo deck](https://raw.githubusercontent.com/JoshKarpel/spiel/main/docs/assets/demo.svg)
 ![The demo deck in "deck view"](https://raw.githubusercontent.com/JoshKarpel/spiel/main/docs/assets/deck.svg)
 
+## Quick Start:
+
+The most basic Spiel presentation looks like this:
+
+```python
+from spiel import Deck, present
+
+deck = Deck(name="Your Deck Name")
+
+@deck.slide(title="Slide 1 Title")
+def slide1():
+    content = """
+    Your content here
+    """
+    return content
+
+if __name__ == "__main__":
+    present(__file__)
+```
+
+And when you run the Python file:
+![Barebones slide](./docs/assets/quickstart1.svg)
```

Comment:
I was playing with these over the last few days - I should really write a contribution guide to describe how to do this...

There's a script at https://github.com/JoshKarpel/spiel/blob/30e676bee4bed388eaaa8901cc41f734b1ecc2d6/docs/generate_screenshots.py#L62-L64 that is hooked up to `pre-commit` (incidentally, make sure to install `pre-commit` and then run `pre-commit install`). The idea is to make sure all the screenshots are always up-to-date and to identify any changes elsewhere that cause those screenshots to change. So, I'd prefer if you could refactor that script so that it can work with a deck that isn't the demo deck, and to generate these screenshots via that script.

Also note how I did the URLs in https://github.com/JoshKarpel/spiel/pull/174/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R24-R25 - that should make them work on PyPI next time I release 🤞🏻 (this `README.md` becomes the PyPI project description).

Reply:
I made some updates in my branch, but I'm failing the pre-commit because "TypeError: SpielApp.__init__() got an unexpected keyword argument 'show_messages'"

Everything looks fine, but it's failing for me in the main pre-commit too, so I don't think it was introduced by my changes. Any ideas what might be going wrong there?

Reply:
Interesting, I can't reproduce, but presumably it's failing inside `generate_screenshots.py`? I suspect the problem is that you might have `spiel` installed via `pip` and it's overriding the local install. I recommend redoing your `poetry install` and making sure you're in `poetry shell`. I should really write some contributing docs...

Reply:
Hmm, yeah it's failing in `generate_screenshots.py`. I wasn't using poetry, just pip install and a virtual environment. I installed poetry, but I'm not super familiar with it or what I should be doing differently. Is it as simple as `poetry shell` and then `git commit`? Or are there other things I should be doing in there?

Reply:
Pretty much! I threw together a quick guide here https://www.spiel.how/contributing/#initial-setup ; feel free to edit that too if it doesn't work as expected.

Reply:
Thanks, that worked!